effekt/jvm/src/test/scala/effekt/ParserTests.scala

```diff
@@ -0,0 +1,58 @@
+package effekt
```

Comment:
@dvdvgt can we delete this file? Now that we have the recursive descent, I believe it is not necessary anymore, no?